declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY   
+include::_attributes/common-attributes.adoc[]
+[id="managing-app-sets-in-non-control-plane-namespaces"]
+= Managing the application set resources in non-control plane namespaces
```

Comment:
I think we should combine them?
cc: @svghadi 

```suggestion
= Managing the applicationset resources in non-control plane namespaces
```

Reply:
@varshab1210 and @svghadi As per this guideline - https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting, we have to keep it separate and we are retaining it as is here.